components/container/index.test.jsx

```diff
@@ -156,8 +158,9 @@ describe("Container view", () => {
 
   it("renders Not Supported if resource loaded is not a container", () => {
     const resourceIri = "http://example.com/resource";
-    mockedDatasetHook.mockReturnValue({
-      data: mockSolidDatasetFrom(resourceIri),
+    mockedContainerHook.mockReturnValue({
+      data: { dataset: mockSolidDatasetFrom(resourceIri) },
```

Comment:
You're creating very specific objects that only has the properties you need for this test - you should instead create mocked objects of models (with functions that can be reused elsewhere).

For example, we could have a factory method that takes the URL of the thing you want, which mocks a thing and dataset (which you must strip the hash-part of the URL, if any) for you based on that one URL.